tests/robot_tests.cpp

```diff
@@ -0,0 +1,53 @@
+#include <pick_ik/robot.hpp>
+
+#include <catch2/catch_all.hpp>
```

Comment:
Consider using `catch2/catch_test_macros.hpp` which is a much lighter weight header that just has the macros we use and doesn't also bring it tons of other things we're probably not using.

Reply:
done